packages/@react-spectrum/table/stories/Table.stories.tsx

```diff
@@ -951,7 +961,7 @@ function AsyncLoadingExample(props) {
     sort({items, sortDescriptor}) {
       return {
         items: items.slice().sort((a, b) => {
-          let cmp = a.data[sortDescriptor.column] < b.data[sortDescriptor.column] ? -1 : 1;
+          let cmp = a.data[sortDescriptor.column!] < b.data[sortDescriptor.column!] ? -1 : 1;
```

Comment:
shouldn't `column` be required? when would you have a sort descriptor but not a column?